pkg/alert/alert.go

```diff
@@ -1,8 +1,177 @@
 package alert
 
-import "fmt"
+import (
+    "fmt"
+    "os"
+    "time"
+    "encoding/json"
+    "io/ioutil"
+    "net/http"
 
-func Stale() error {
-	fmt.Println("FIXME!")
+    "github.com/anynines/a9s-aha-cli/pkg/notify"
+
+    log "github.com/sirupsen/logrus"
+)
+
+type Content struct {
+    Idea []struct {
+        Name string `json:"reference_num"`
+        Workflow_status struct {
+            Name string `json:"name"`
+        } `json:"workflow_status"`
+    } `json:"ideas"`
+}
+
+type Req struct {
+    Comments struct {
+        Updated_at string `json:"updated_at"`
+        Name string `json:"name"`
+    } `json:"idea"`
+}
+
+func Stale(notifyFlag bool) (error) {
+    var status Content
+    var updated_at Req
+    var msg string
+
+    err := CheckEnvVariables(notifyFlag)
+    if err != nil {
+        return err
+    }
+
+    log.Info("Query aha.io idea list: ")
```

Comment:
We should not log anything by default I would say. If someone wants more details, he can always use the verbose flag.